packages/functional-tests/tests/react-conversion/signInRelyingParties.spec.ts

```diff
@@ -40,24 +40,14 @@ test.describe('severity-1 #smoke', () => {
 
     await expect(connectAnotherDevice.fxaConnected).toBeEnabled();
     await connectAnotherDevice.notNowButton.click();
-
-    await expect(page).toHaveURL(/settings/);
-
-    // Normally we wouldn't need this delay, but because we will be
-    // disconnecting the sync service, we need to ensure that the device
-    // record and web channels have been sent and created.
-    await page.waitForTimeout(1000);
+    await expect(page).toHaveURL(/settings/, { timeout: 1000 });
 
     await settings.disconnectSync(credentials);
 
-    // See above, we need to wait for disconnect to complete
-    await page.waitForTimeout(1000);
-
     // confirm left settings and back at sign in
-    expect(page.url()).toBe(`${target.contentServerUrl}/`);
+    await page.waitForURL('**/signin', { timeout: 1000 });
```

Comment:
Just curious if there's a reason not to use the default timeout here?

Reply:
All these steps take extra time to load the page and hence were failing, so I had tio remove the `waitForTimeout()` and use the `options` with a conditional wait.